src/backend/distributed/executor/multi_utility.c

```diff
@@ -296,9 +304,9 @@ multi_ProcessUtility(Node *parsetree,
 									  "move all tables.")));
 		}
 	}
-	else if (!schemaNode)
+	else
```

Comment:
The fact that there is only one such if-block for the case where !EnableDDLPropagation seemed a arbitrary, so I wanted to leave space for more. Added a comment elaborating on our behaviour when citus.enable_ddl_propagation is off right after the else.